python/llm/example/langchain/native_int4/docqa.py

```diff
@@ -31,16 +31,14 @@
 from langchain.callbacks.manager import CallbackManager
 from langchain.callbacks.streaming_stdout import StreamingStdOutCallbackHandler
 
-from bigdl.llm.langchain.llms import BigdlNativeLLM
-from bigdl.llm.langchain.embeddings import BigdlNativeEmbeddings
-
+from bigdl.llm.langchain.llms import LlamaLLM
+from bigdl.llm.langchain.embeddings import LlamaEmbeddings
 
 
 def main(args):
     
     input_path = args.input_path 
     model_path = args.model_path 
-    model_family = args.model_family
```

Comment:
in the example, I think we still need this argument to support different models.

Reply:
also for other examples.